### ISSUE TITLE
Released version of Semigroups does not compile: libsemigroups/.VERSION is missing?

### ISSUE THREAD

**Issue by wilfwilson** [semigroups/Semigroups#740]
When running `./autogen.sh` or `./configure` in the released version of the Semigroups package (e.g. with `v3.4.1`), it fails because it cannot determine that the libsemigroups version is new enough. The `prerequisites.sh` script fails for the same reason.
```
[...omitted...]
checking for libsemigroups/.VERSION... no
checking for libsemigroups/etc/version-number.sh... yes
checking libsemigroups version... 1.dev
configure: error: libsemigroups version 1.3.2 or higher is required
```
This was spotted by the automated GAP testing of packages. Thanks @alex-konovalov.

This could be fixed by including the appropriate `libsemigroups/.VERSION` file in the release.

**Comment by wilfwilson** [comment]
By the way, does the `autogen.sh` file need to be run on the released version? If not, as I presume, then it probably shouldn't be included with the release.

**Comment by wilfwilson** [comment]
In fact `prerequisites.sh` shouldn't be included in the release either.

Note that GAP packages in the GAP CI get compiled via GAP's `bin/BuildPackages.sh` script, which first runs `prerequisites.sh` (if present), then `autogen.sh` (if present), then `configure` (if present) and `make`.

**Comment by james-d-mitchell** [comment]
Thanks @wilfwilson i seem to have messed up the release somehow, I’ll attempt to resolve this asap

**Comment by alex-konovalov** [comment]
@james-d-mitchell thanks, hope this will be soon!

**Comment by james-d-mitchell** [comment]
@alex-konovalov @wilfwilson this is resolved by v3.4.2

**Comment by wilfwilson** [comment]
Thanks @james-d-mitchell
